### PR TITLE
Adding additional information on how to include 3rd-party JS libraries

### PR DESCRIPTION
**Pull request by eduard13**
## Purpose of this pull request

This pull request (PR) describes how a 3rd party JS Library can be included and used within the entire website.

## Affected DevDocs pages

- https://devdocs.magento.com/guides/v2.2/javascript-dev-guide/javascript/js-resources.html
- https://devdocs.magento.com/guides/v2.3/javascript-dev-guide/javascript/js-resources.html

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: guides/v2.2/javascript-dev-guide/javascript/js-resources.md [hunk 1]
```diff
@@ -136,6 +136,74 @@ To be available for the entire Magento instance, RequireJS library is included i
...
+    },
+    shim: {
+        slick: {
+            deps: ['jquery']
+        }
+    }
+};
+```
+
+After all these steps, clear the cache and deploy the files.
```
Comment (rogyar) [inline guides/v2.2/javascript-dev-guide/javascript/js-resources.md]: Please, clarify this part slightly
```suggestion
After all these steps, clear the cache and perform the static content deployment
```

**Comment by rogyar** [comment]
Hi @eduard13. Thank you for your collaboration. From a technical perspective, your point is correct. 
I would put a new library into a custom module instead of a custom theme since it's much easier to disable/remove a module than editing the theme. 
However, both ways are working and commonly used.

File: guides/v2.2/javascript-dev-guide/javascript/js-resources.md (hunk 1 above)
Comment (eduard13) [reply guides/v2.2/javascript-dev-guide/javascript/js-resources.md]: Thank you for the suggestion, just updated it.

**Comment by eduard13** [comment]
Hi @rogyar, thank you for the review.

**Comment by dobooth** [comment]
running tests

**Pull request merged**